compiler/rustc_abi/src/canon_abi.rs

```diff
@@ -79,6 +82,7 @@ impl fmt::Display for CanonAbi {
                 X86Call::Vectorcall => ExternAbi::Vectorcall { unwind: false },
                 X86Call::Win64 => ExternAbi::Win64 { unwind: false },
             },
+            CanonAbi::Custom => ExternAbi::Custom,
```

Comment:
Up?

ABIs valid on all targets should always precede all other ABIs. Same place in the ordering you put the variant in originally is fine. I realize that the ExternAbi listing isn't well-ordered but I'd like to preserve this much.

Reply:
Thanks!